src/components/elements/alert/alert.tsx

```diff
@@ -0,0 +1,16 @@
+import { Alert as DSAlert } from "@lifesg/react-design-system/alert";
+import { TestHelper } from "../../../utils";
+import { IGenericFieldProps } from "../../frontend-engine";
+import { IAlertSchema } from "./types";
+
+export const Alert = (props: IGenericFieldProps<IAlertSchema>) => {
+	// =============================================================================
+	// CONST, STATE, REF
+	// =============================================================================
+	const { id, schema } = props;
+
+	// =============================================================================
+	// RENDER FUNCTIONS
+	// =============================================================================
+	return <DSAlert id={id} data-testid={TestHelper.generateId(id, "alert")} {...schema} />;
```

Comment:
is it possible to parse HTML from string? we will want to serve HTML text from JSON